appcircle_npm_yarn/1.0.1/component.yaml

```diff
@@ -0,0 +1,27 @@
+platform: Common
+buildPlatform:
+displayName: npm/Yarn Commands
+description: "Runs npm/yarn with the given commands."
+docsUrl: https://docs.appcircle.io/workflows/react-native-specific-workflow-steps#npmyarn-commands
+requiredComponents: "appcircle_git_clone,appcircle_node_install"
+precedingComponents: 
+followingComponents: "appcircle_export_build_artifacts,appcircle_ios_build_sign,appcircle_ios_build_simulator,appcircle_ios_build_for_testing"
```

Comment:
> @boztopuz Since this review is very repetitive, I will not give the same feedback in the rest, but can you provide the checks for all of them?

I mentioned the above situation in the previous review, but since the `export_build_artifacts` for which I did not add a comment were not corrected, I am adding them again for each one.
=> There is no any dependencies with `appcircle_export_build_artifacts`